uuid.go

```diff
@@ -213,6 +213,13 @@ func (u *UUID) SetVariant() {
 	u[8] = (u[8] & 0xbf) | 0x80
 }
 
+// MarshalJSON implements the json.Marshaler interface.
+// The encoding is the same as returned by String except the quotation marks are added around.
+func (u UUID) MarshalJSON() (data []byte, err error) {
+	data = []byte(fmt.Sprintf("\"%x-%x-%x-%x-%x\"", u[:4], u[4:6], u[6:8], u[8:10], u[10:]))
```

Comment:
Can we just call MarshalText() here (or String()) and add the quotation marks around? And similarly in UnmarshalJSON()?